docs/writing-docs/doc-blocks.md

```diff
@@ -0,0 +1,370 @@
+
+<div style="background-color:#F8FAFC">
+TODO:add image per screenshot requirements (mention of a image in the SB 6.0 doc (needs to be vetted))
+</div>
+
+```js
+
+export default {
+  title: 'CustomDescription'
+  parameters: {
+    docs: { 
+        description: { 
+            component: 'some component _markdown_' 
+        } 
+    },
+  }
+};
+
+export const WithStoryDescription = Template.bind({});
+WithStoryDescription.parameters = {
+  docs: { 
+      description: { 
+          story: 'some story **markdown**' 
+        } 
+    },
+};
+```
+
+#### MDX
+
+In MDX, the `Description` shows the component’s description using the same heuristics as the DocsPage. It also accepts a markdown parameter to show any user-provided Markdown string.
+
+```js
+
+import { Description } from '@storybook/addon-docs/blocks';
+import dedent from 'ts-dedent';
+import { Button } from './Button';
+
+<Description of={Button} />
+<Description markdown={dedent`
+  ## Custom description
+
+  Insert fancy markdown here.
+`}/>
+```
+
+### Story
+
+Stories (component examples) are the basic building blocks in Storybook. In Storybook Docs, stories are rendered in the `Story` block.
+
+#### DocsPage
+
+In DocsPage a `Story` block is generated for each story in your [CSF] file, wrapped with a `Preview` wrapper that gives it a toolbar on top (in the case of the first “primary” story) and a source code preview underneath.
+
+#### MDX
+<div>
+TODO: ask tom about Preview link for clarity.
+</div>
+
+In MDX, the `Story` block is not only a way of displaying stories, but also the primary way to define them. Storybook looks for `Story` instances with the `name` prop, either defined at the top level of the document, or directly beneath a [Preview](locate-preview) block defined at the top level:
+
+```js
+import { Story } from '@storybook/addon-docs/blocks';
+import { Button } from './Button';
+
+export const Template = (args) => <Button {...args} />;
+
+<Story name="Basic" args={{ label: ‘hello’ }}>
+  {Template.bind({})
+</Story>
+```
+
+You can also reference existing stories in Storybook by ID:
+
+```js
+
+import { Story } from '@storybook/addon-docs/blocks';
+
+<Story id="some-component--some-name" />
+```
+
+#### Inline rendering
+
+<div>
+TODO: ask tom/michael about the preview iframe. as we've been calling it canvas so far and now move to preview.
+And also inline rendering support location
+</div>
+
+In Storybook’s Canvas, all stories are rendered in the [Preview iframe] for isolated development. In Storybook Docs, when [inline rendering is supported by your framework](locate-link), inline rendering is used by default for performance and convenience. However, you can force iframe rendering with `docs: { inlineStories: false }` parameter, or `inline={false}` in MDX.
+
+
+### Preview
+
+<div>
+TODO: ask tom about Source link for clarity, as me i know that the source shows ad hoc the story file content. But another reader might not know.
```

Comment:
I think Source just links back up to the source doc-block. Does that not make sense to you?

Reply:
Linked it per feedback. And yes it makes sense to point for that section